### DISCUSSION TITLE
posts/2024/04/field-essay-50/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#51] [General]
# posts/2024/04/field-essay-50/

或许，举办第一次以“神明”为主题的座谈会起初就不是为疑问提供一个准确的回答，也从未期待来自村内与村外的村民因此便产生了紧密的联系，而是在讨论问题与讲述故事的过程中发现兴趣的产生、角色身份的差异、“民间”与“学院”互动时产生的碰撞……
组织一场聚集来自不同社区、身份背景各异的村民的座谈会不是一件容易的事情，从人员的挑选、邀请到互动，无一不在寻求一股“撬动社区的力量”。

https://guozheng.rbind.io/posts/2024/04/field-essay-50/

**Comment by ghost** [comment]
屈原不是温州人，其故里在现湖北省秭归县。红卫社区的“温州茅竹水仙大王”，这里的“温州”指的是浙江省温州市永嘉县的茅竹水仙，包括温州大殿、温州吕大殿、敕赐温州、温州茅竹、茅竹，这些称号都是一样的，都指向同一地点，不同称呼，全代表着供奉永嘉的茅竹水仙大王，只是西埠村水仙公直接省略了温州二字，简称茅竹水仙大王而已。
在明清《指南正法》、《顺风相送》航海针路薄记载着“茅竹水仙”称呼，流传至今。元王敬方《褒封水仙记》记载浙江永嘉茅竹水仙称“国朝漕运，为事最重，故南海诸神，有功于漕者皆得祀。......并居永嘉茅竹里，庙在焉，故俗称“茅竹水仙”云。”至此，茅竹水仙成为道教神仙体系中的一部分。

**Reply by ghost** [reply]
硇洲岛上的俗称妈祖为观音菩萨，祖妈为天后圣母。在中国民间的称呼中，天后被称为娘妈、妈祖，意为娘妈之祖。这便是硇洲和外地叫法不一样的原因，容易搞混。

**Reply by ghost** [reply]
现在会主广福康车二相每年都在诞辰之后坐船前往徐闻，因为当年从从硇洲躲避海匪搬迁到徐闻定居下的人民依旧有份服侍该神灵。而会主广德康车二相他们请过去徐闻才会过去了，不请的话就不坐船去徐闻了。
相传金兰会主广福广德康车四相为硇洲最古老、最大的神灵，在远古时代便有了，也被称为太祖。每年游神一定得他们先游神，其它村庄的神灵才能跟着在后面游。其它神灵光彩便会请他们去作证。
在硇洲岛上来说广福康皇仁恩大帝、会主广福烈雷车大元帅、会主广德康皇仁恩大帝、会主广德烈雷车大元帅为最大、最重要的神灵；对于全国来说关公、天后供奉更多、更大；在岛上，广福、广德康皇是比关公、天后神涵大的

**Reply by residualsun1** [reply]
> 屈原不是温州人，其故里在现湖北省秭归县。红卫社区的“温州茅竹水仙大王”，这里的“温州”指的是浙江省温州市永嘉县的茅竹水仙，包括温州大殿、温州吕大殿、敕赐温州、温州茅竹、茅竹，这些称号都是一样的，都指向同一地点，不同称呼，全代表着供奉永嘉的茅竹水仙大王，只是西埠村水仙公直接省略了温州二字，简称茅竹水仙大王而已。
在明清《指南正法》、《顺风相送》航海针路薄记载着“茅竹水仙”称呼，流传至今。元王敬方《褒封水仙记》记载浙江永嘉茅竹水仙称“国朝漕运，为事最重，故南海诸神，有功于漕者皆得祀。......并居永嘉茅竹里，庙在焉，故俗称“茅竹水仙”云。”至此，茅竹水仙成为道教神仙体系中的一部分。

哇，我感觉这里面已经不止是《全粤村情》的记载了呀，似乎是一些历史研究的内容。你对岛屿的神明了解也很多哇！

> 硇洲岛上的俗称妈祖为观音菩萨，祖妈为天后圣母。在中国民间的称呼中，天后被称为娘妈、妈祖，意为娘妈之祖。这便是硇洲和外地叫法不一样的原因，容易搞混。

我们开座谈会这晚，余校长和陈老师都认为西埠村的村民记错了，后来谭道士也告诉我村民们完全叫错了。我想可能有不少村民在口口相传，或者其他啥原因中，出现了别的叫法，总的，他们的理解在一些研究本地文化的爱好者和道士们看来都是有误的。

> 现在会主广福康车二相每年都在诞辰之后坐船前往徐闻，因为当年从从硇洲躲避海匪搬迁到徐闻定居下的人民依旧有份服侍该神灵。而会主广德康车二相他们请过去徐闻才会过去了，不请的话就不坐船去徐闻了。
相传金兰会主广福广德康车四相为硇洲最古老、最大的神灵，在远古时代便有了，也被称为太祖。每年游神一定得他们先游神，其它村庄的神灵才能跟着在后面游。其它神灵光彩便会请他们去作证。
在硇洲岛上来说广福康皇仁恩大帝、会主广福烈雷车大元帅、会主广德康皇仁恩大帝、会主广德烈雷车大元帅为最大、最重要的神灵；对于全国来说关公、天后供奉更多、更大；在岛上，广福、广德康皇是比关公、天后神涵大的

很想知道这些说法是哪来的嘞？看着更像是故事一样的说法，不像是《全粤村情》里的内容（PS虽然我还没看过）

**Reply by ghost** [reply]
我以前也是经常听我奶奶讲说硇洲岛的神明故事，略有了解而已！

**Reply by ghost** [reply]
她能讲述硇洲很多关于神灵的故事，她是重建红卫水仙宫的头人之一。听她讲述，硇洲现在的广福是在五班北赛村雕刻的，而广德是在二班应凡村雕刻的。当年文化大革命基本把岛上的庙宇和神像都破坏了一遍，能保留下来的很少，所以现在我们看到基本上都是后来重建和重刻的产物，唯独南港的真相被藏起来神像才保留了下来。相传老泰山和二泰山本是一对，在古代硇洲中村范围供奉的，后来老泰山下去了塘尾村，二泰山则在那娘村（现周屋村），后来南港入会后渐渐的变成了在周屋谭巴利和南港之间轮流敬奉了。

**Reply by residualsun1** [reply]
@3174481033 

> 当年文化大革命基本把岛上的庙宇和神像都破坏了一遍，能保留下来的很少，所以现在我们看到基本上都是后来重建和重刻的产物，唯独南港的真相被藏起来神像才保留了下来。

我也听老人说广兆泰山的神像也是一直保留下来的，文革那会儿被调包了。

> 她能讲述硇洲很多关于神灵的故事，她是重建红卫水仙宫的头人之一。

如果以后再回硇洲，如果可以的话，我希望能有机会听听她讲讲故事~

真的很感谢你的分享😄，让我们了解到了许多丰富又有点意料之外的信息。对了，这篇座谈会的回顾我前两天好好大幅修改了一遍，配上了更合适的图片，如果你有兴趣的话可以到我们伙伴的公众号（[文章](https://mp.weixin.qq.com/s/4N3FGGITosLDWC_w9nt6Dw)）看看。

另，我有点好奇你是了解到我的网站网址的？我一般都没有对外公开网址，而且谷歌和百度似乎也没有收录我的网址，从没想到会有陌生人访问到这里。第一次收到你关于天后介绍的留言，我是又意外又欣喜。

**Reply by ghost** [reply]
没有机会了，她前两年已经走了

**Reply by ghost** [reply]
可能是由于我用国际版浏览器才刷到的，我也是无意间点进去看到你的博客的，哈哈哈哈哈哈

**Reply by ghost** [reply]
公众号照片里面戴黑色眼镜，穿白色衣服那个是你来的吗😂

**Reply by residualsun1** [reply]
> 没有机会了，她前两年已经走了

抱歉🙏😔。我想起第一次看你留言的时候，和身边的伙伴提起这件事，他后来在自己的理发日活动回顾的文章里写了这么一段话：

> 有个老人前来理发，打听之后才知道是前两天提到跟一位研究贝类的老师一起共事的人，攀谈过后，了解到他三十五岁前是靠出海为生，曾经出租自己的船跟我们学校的老师一起去海上采集贝类，也曾跟着另外一位老师一起做珍珠贝人工养殖的工作，提起跟我学校老师的渊源，他很是兴奋，我们说起他有一本前面提到蔡老师给他的赠书，现如今“流落”在黑石屿，他提出想看看。于是乎，这本年纪有二十三年的老书又回到它主人手中，看着扉页的作者的字迹，好像有种穿越的感觉，仿佛是上天的指引，在时空的裂隙中，将二十几年前的记忆重新重新拾掇起来。这本书是联系我们三方的工具，是时间的使者，见证了二人浓厚的感情，而以后它也将留在黑石屿民宿，不仅能让更多后来者可以读到到作者的作品，还能感受到作者与受赠者的情谊。情到此处，我不禁想起一些事情，类似这种作者与受赠者的情感载体转移，即书籍本身的转移，现实生活中有很多很多，譬如一些高校做研究的老人离世后，生前的书籍可能作为赠品送到了图书馆，可能被卖到了废品站，也可能流落在某个二手书商贩的仓库角落。作者和受赠者的故事不被后人所知，后人只知道书籍的内容，而书籍作为情感纽带的作用在受赠者死去那一刻已经消散。**我想到了为何要发掘历史，它可能不是宏观的大是大非，也许它只是一段时间内两个人的情感交集，无关对错，它在历史的长河中是客观存在的，要是能将某件事情联系起来，这个发掘过程和结果就是有意义的**。

（现在再看，觉得他写得真好！）略过一部分，承接下面这段⬇

> 后文–与国政的讨论（关于神明）
下午在跟国政聊天得知他写的博客有人回复他了，两个完全没有交集的人，竟开始讨论起西埠的神明来历，我看到了一个以往做过研究的人在给试图靠近这段历史的人的帮助，我的内心好像突然被敲打了一下，博客在现如今作为一种冷门的平台，居然因为神明而将两人的世界联系起来。而他们讨论中出现的一个名字也让我很感兴趣，一个神明的名字——康皇，这不是我们老家供奉的神吗，怎么这里也会有，国政说不同地方的神明来历可能不同，它历史上的原型可能也有不同说法，于是我就很想知道我老家那边的神明的来历，评论区中一本叫《全粤村情》似乎能解答我的疑问，但是搜遍网上都找不到我老家的那一卷，失落极了，但是后面国政在一个网站算是找到了我们老家那边神明在历史上的来历，虽然不清楚完整的故事，但是也算达成目标。

我突然觉得你将你奶奶讲述的故事告知于我，我们似乎也因此“联结”了起来，这是一种很神奇的感觉，不仅应当对你表达谢意，也应该对你奶奶表示谢意。

> 公众号照片里面戴黑色眼镜，穿白色衣服那个是你来的吗😂

对，看起来非常呆滞😵‍💫。

**Reply by ghost** [reply]
没事，那些已经是过去的事情了。有很多东西跟随老一辈的去世已经开始在慢慢失传了，觉得书籍记载挺重要的，要不后人再也没有机会了解其中的意义了。
你可以搜索下北京畅想之星公众号，《湛江村庄简史》也许能帮助你，不过也只能看一部分，只能推荐给图书馆购买才可看，特别烦人。
一看就像文科生了😂，哈哈哈哈哈哈，你为什么会读这个专业的呢？

**Reply by residualsun1** [reply]
图片仅供参考，现实的人还是有点不同的😂。

我读的社会学专业，不过当时报志愿的时候也不懂这个专业，在专业组里分数匹配到了这个专业就读下来啦。不过我现在觉得很幸运很幸运能读到这个专业，正是这个专业、这个专业所在的学校和它背后的老师，让过去的“我”成为了今天的“我”: )。.

**Reply by residualsun1** [reply]
@3174481033 你好呀，端午节快到了，硇洲应该很快也有爬龙仪式了，不知道你对爬龙是否了解呢 ;-)？

**Comment by CyrusYip** [comment]
https://github.com/residualsun1/Residualsun/discussions/51#discussioncomment-9313440

@residualsun1 你的网站已经被谷歌和必应采集了，搜 `site:guozheng.rbind.io` 就能看到。百度只收录了主页。

**Reply by residualsun1** [reply]
没想到还是给收录了，好吧……希望我的同学不会搜到我的网站😂。

**Reply by CyrusYip** [reply]
谷歌有禁止收录的教程：[Block Search Indexing with noindex | Google Search Central  |  Documentation  |  Google for Developers](https://developers.google.com/search/docs/crawling-indexing/block-indexing)，其他搜索引擎也有相关教程，搜索「某某搜索引擎 block index」就能找到了。

**Reply by residualsun1** [reply]
贴心教程😄，已码住。
